src/tests/CMakeLists.txt

```diff
@@ -53,16 +53,17 @@ find_package(Boost 1.57.0 REQUIRED COMPONENTS unit_test_framework)
   list(APPEND test_cases test_jobtopology)
   list(APPEND test_cases test_dipoledipoleinteraction)
   list(APPEND test_cases test_populationanalysis)
-list(APPEND test_cases test_gaussian)
-list(APPEND test_cases test_orca)
-list(APPEND test_cases test_nwchem)
-list(APPEND test_cases test_dftengine)
-list(APPEND test_cases test_bsecoupling)
-list(APPEND test_cases test_rate_engine)
+  list(APPEND test_cases test_gaussian)
+  list(APPEND test_cases test_orca)
+  list(APPEND test_cases test_nwchem)
+  list(APPEND test_cases test_dftengine)
+  list(APPEND test_cases test_bsecoupling)
+  list(APPEND test_cases test_rate_engine)
+  list(APPEND test_cases test_eigencuda)
```

Comment:
What will the test do if `USE_CUDA` is off?

Reply:
It will fail ;) Thanks!